src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -2416,18 +2427,7 @@ window.qBittorrent.DynamicTable ??= (() => {
                 row.full_data.remaining = (row.full_data.size * (1.0 - (row.full_data.progress / 100)));
         },
 
-        setupTr: function(tr) {
-            tr.addEventListener("keydown", function(event) {
-                switch (event.key) {
-                    case "ArrowLeft":
-                        qBittorrent.PropFiles.collapseFolder(this._this.getSelectedRowId());
-                        return false;
-                    case "ArrowRight":
-                        qBittorrent.PropFiles.expandFolder(this._this.getSelectedRowId());
-                        return false;
-                }
-            });
-        }
+        setupCommonEvents: () => {}
```

Comment:
Bulk rename table does not make any use of common events from what I could tell.